packages/manager/modules/nasha/src/dashboard/partition/snapshots/snapshots.template.html

```diff
@@ -4,10 +4,23 @@
         data-description="{{:: 'nasha_dashboard_partition_snapshots_description' | translate }}"
     ></oui-header>
     <h4 data-translate="nasha_dashboard_partition_snapshots_subtitle"></h4>
+    <p class="d-flex flex-column">
+        <strong
+            data-translate="nasha_dashboard_partition_snapshots_count_title"
+        ></strong>
+        <span
+            data-translate="nasha_dashboard_partition_snapshots_count"
+            data-translate-values="{
+                count: $ctrl.customSnapshots.length,
+                max: $ctrl.MAX_CUSTOM_SNAPSHOT,
```

Comment:
Suggestion: Can we move the calculation {{ count }}/{{ max }}to controller ? 
  data-translate-values="{
                maximum: $ctrl.maximum,}"

Reply:
This is not a calculation but a display of information. Would you prefer I use this.$translate.instant()?